staging/src/k8s.io/api/discovery/v1alpha1/types.go

```diff
@@ -59,21 +59,26 @@ type EndpointSlice struct {
 type AddressType string
 
 const (
-	// AddressTypeIP represents an IP Address. Inclusive of IPv4 and IPv6
-	// addresses.
+	// AddressTypeIP represents an IP Address.
+	// This address type has been deprecated and has been replaced by the IPv4
+	// and IPv6 adddress types. New resources with this address type will be
+	// considered invalid. This will be fully removed in 1.18.
+	// +deprecated
```

Comment:
do we really need to leave a deprecation message in api doc? I mean it is alpha state, what is the guidance here?

Reply:
I think it's better to be safe here, but also curious what api-reviewers say.

Reply:
Looks like deprecation is the thing to do here (if I'm understanding comments below correctly).